src/faber/tools/python.py

```diff
@@ -72,23 +72,42 @@ def __init__(self, name='python', command=None, version='', features=()):
         # Now determine all the flags we may need to compile C / C++ extensions
         self.include = include(self.check_sysconfig('get_python_inc()'))
         platform = self.check_python('import platform; print(platform.system())')
+        impl = self.check_python('import platform; print(platform.python_implementation())')
+        prefix = self.check_python('import sys; print(sys.prefix)')
         if platform == 'Windows':
-            version = self.check_python('import sys; print("%d%d"%sys.version_info[0:2])')
-            prefix = self.check_python('import sys; print(sys.prefix)')
-            self.libfile = join(prefix, 'libs', 'python{}.lib'.format(version))
-            self.libpath = join(prefix, 'libs')
-            self.lib = 'python' + version
+            if impl == 'CPython':
+                version = self.check_python('import sys; print("%d%d"%sys.version_info[0:2])')
```

Comment:
I think you are looking for `sysconfig.get_config_var('py_version_nodot')` since on python 3.10 this will become `3_10` not `310` (at least that is the direction from python/cypthon#20333)

Reply:
Thanks for the tip !